Source/WebKit/UIProcess/WebProcessProxy.cpp

```diff
@@ -414,6 +414,11 @@ WebProcessProxy::~WebProcessProxy()
     while (m_numberOfTimesSuddenTerminationWasDisabled-- > 0)
         WebCore::enableSuddenTermination();
 
+#if PLATFORM(COCOA) && !ENABLE(STREAMING_IPC_IN_LOG_FORWARDING)
```

Comment:
I think this should be `ENABLE(LOGD_BLOCKING_IN_WEBCONTENT) && ENABLE(STREAMING_IPC_IN_LOG_FORWARDING)` to match `addMessageReceiver`.
Maybe we should move this in `platformDestroy()` in WebProcessProxyCocoa.mm too.

Reply:
That's a good catch! I am moving this to `platformDestroy()` in the upcoming patch.

Thanks for reviewing!